daemons/schedulerd/pacemaker-schedulerd.h

```diff
@@ -15,5 +15,8 @@
 
 extern pcmk__output_t *logger_out;
 extern struct qb_ipcs_service_handlers ipc_callbacks;
+extern GHashTable *schedulerd_handlers;
+
+void schedulerd_register_handlers(void);
```

Comment:
I'd prefer for all the IPC header contents to go in `<daemon>_ipc.h`. I think ideally each `.c` file would get its own respective `.h` file, and the main `<daemon>.h` header file would collect them all as well as whatever globals and inlines can't go into more specific files. However, `pacemaker-schedulerd.h` is so small that I don't care at the moment.

As an aside, I don't think we need to include `crm/common/scheduler.h` in this file.